tools/arm_compiler/build_raspberry_pi.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -e
+
+# To be able to build any of the python packages, it is assumed that
+# you have a python directory in /usr/include/arm-linux-gnueabihf/.
+
+# To install the architecture includes for python on ubuntu trusty;
+# run:
+#  sudo dpkg --add-architecture armhf
+#  echo "deb [arch=armhf] http://ports.ubuntu.com/ trusty main universe" \
+#    sudo tee -a /etc/apt/sources.list.d/armhf.list
+#  # Ignore errors about missing armhf packages in other repos.
+#  sudo apt-get update
+#  sudo apt-get install libpython-all-dev:armhf
```

Comment:
I had to use aptitude instead of apt-get on my Goobuntu machine, since otherwise I saw:

Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libpython-all-dev:armhf : Depends: libpython-dev:armhf (= 2.7.5-5ubuntu3) but it is not going to be installed
                           Depends: libpython2.7-dev:armhf but it is not going to be installed
E: Unable to correct problems, you have held broken packages.